26-register_publication/274-creator_navigates_to_the_reference_tab_and_selects_contribution_to_journal.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Creator navigates to the Reference tab and selects Contribution to Journal
+
+  @274
+  Scenario: Creator navigates to the Reference tab and selects Contribution to Journal
+    Given that the Creator opens their Publication and navigates to the Reference tab
+    When they select Contribution to Journal as Reference Type
+    And they see a dropdown of potential Reference Subtypes consisting of:
+      | Journal article      |
+      | Short communication  |
+      | Letter to the Editor |
+      | Book review          |
+      | Editoral             |
+      | Corrigendum          |
+    And they see mandatory fields for:
+      | Search-box for Journal |
+    And they see optional fields for:
+      | Volume         |
+      | Issue          |
+      | Pages from     |
+      | Pages to       |
+      | Article number |
+    And they see a disabled field with DOI if the Publication has a DOI
+    And they see a optional field for Peer-review if Journal article is selected as Reference Subtype
```

Comment:
er vel ikke heeeelt innafor å slenge på `if`s på en `And` 😬

Reply:
kommer til å dele opp dette til flere stories i stedet, ref kommentaren til Terje